src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -176,17 +177,37 @@ public async Task RefreshAsync()
             }
         }
 
-        public async Task<bool> TryRefreshAsync()
+        public async Task<bool> TryRefreshAsync(ILogger logger)

```

Comment:
Since we're breaking the interface it's probably a good idea to add cancellation token.

Reply:
Will add cancellation token in a separate PR.